_maps/readme.md

```diff
@@ -4,29 +4,25 @@
 
 The list of items that are modular requiring adding/modifying to /tg/ maps:
 
-- Positronic Alert Consoles (inside Robotics area)
-- Command Vendor (inside Meeting area)
-- LustWish Vendor (inside Recreational/Dormitory area)
 - Barbershop Room (including the vendors, chairs, mirrors, and more)
-- NanoTrasen Consultant's Office
-- Blueshield Office
 - Cryopods (inside Recreational/Dormitory area)
-- Stasis Beds to Stasis Pods
-- Armory/Gun Change
-- Prison Change
```

Comment:
Seems like you forgot to move over this one, our prisons are often different from /tg/'s, and considering the amount of people that worked on bringing them back every time they were forgotten about, I don't think that it's warranted to remove them from that list.

Reply:
Our current maps don't have any different prisons currently, so I have nothing to take over

Reply:
They had custom prisons up until 2 days ago, specifically on Delta I believe. And Icebox, until second last reset